scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -982,38 +961,39 @@ public void jobTaskStarted(final InternalJob job, final InternalTask task,
             public Void executeWork(Session session) {
                 long jobId = jobId(job);
 
-                String jobUpdate = "update JobData set status = :status, "
-                        + "startTime = :startTime, numberOfPendingTasks = :numberOfPendingTasks, "
-                        + "numberOfRunningTasks = :numberOfRunningTasks where id = :jobId";
+                String jobUpdate = "update JobData set status = :status, " +
+                    "startTime = :startTime, numberOfPendingTasks = :numberOfPendingTasks, " +
+                    "numberOfRunningTasks = :numberOfRunningTasks where id = :jobId";
 
                 JobInfo jobInfo = job.getJobInfo();
 
-                session.createQuery(jobUpdate).setParameter("status", jobInfo.getStatus()).setParameter(
-                        "startTime", jobInfo.getStartTime()).setParameter("numberOfPendingTasks",
-                        jobInfo.getNumberOfPendingTasks()).setParameter("numberOfRunningTasks",
-                        jobInfo.getNumberOfRunningTasks()).setParameter("jobId", jobId).executeUpdate();
+                session.createQuery(jobUpdate).setParameter("status", jobInfo.getStatus())
+                        .setParameter("startTime", jobInfo.getStartTime())
+                        .setParameter("numberOfPendingTasks", jobInfo.getNumberOfPendingTasks())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "numberOfPendingTasks" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)